deployment-platforms/zeit-now/README.md

```diff
@@ -3,4 +3,4 @@
 [![Deploy with ZEIT Now](https://zeit.co/button)](https://zeit.co/new/project?template=https://github.com/prisma/prisma-examples/tree/master/deployment-platforms/zeit-now)
 
 
-[Deployment guide](https://www.prisma.io/docs/guides/deployment/deploying-to-zeit-now)
+[Deployment guide](#link-to-docs)
```

Comment:
This is a strange change

Reply:
Again not me

Reply:
cc @nikolasburk was probably only changed in the readme and not in the template